lib/validate-json.js

```diff
@@ -13,33 +13,33 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const { Validator, ValidationError } = require('jsonschema');
+const { Validator } = require('jsonschema');
 const scheduleInputSchema = require('./schemas/support-shift-scheduler-input.schema.json');
 const scheduleOutputSchema = require('./schemas/support-shift-scheduler-output.schema.json');
 
 /**
  * Validate JSON input for scheduler
  * @param  {array}   json   JSON input object
- * @return {object}         jsonschema Validator object
+ * @return {Promise<object>}         jsonschema Validator object
  */
-async function validateJSONScheduleInput(json = {}) {
+async function validateJSONScheduleInput(json = []) {
 	let validator = new Validator();
 	return validator.validate(json, scheduleInputSchema, {
 		throwError: true,
-		lestedErrors: true,
+		nestedErrors: true,
 	});
 }
 
 /**
  * Validate JSON output for scheduler
  * @param  {array}   json   JSON output object
- * @return {object}         jsonschema Validator object
+ * @return {Promise<object>}         jsonschema Validator object
  */
-async function validateJSONScheduleOutput(json = {}) {
+async function validateJSONScheduleOutput(json = []) {
 	let validator = new Validator();
 	return validator.validate(json, scheduleOutputSchema, {
 		throwError: true,
-		lestedErrors: true,
+		nestedErrors: true,
```

Comment:
This means that nested errors will now actually be logged - I'm pretty sure that's fine/intended but just in case let me know and I can remove this